src/streamlink/plugins/wwenetwork.py

```diff
@@ -37,109 +38,133 @@
+        "live": "https://dce-frontoffice.imggaming.com/api/v4/event/{0}",
+    }
+
+    _API_HEADERS = {
+        "x-api-key": "857a1e5d-e35e-4fdf-805b-a87b6f8364bf",
+        "realm": "dce.wwe",
+    }
+
+    def _login(self, email, password):
+        success, data = self.session.http.post(
+            self._API_LOGIN_URL,
+            acceptable_status=(200, 401, 404),
+            json={"id": email, "secret": password},
+            headers=self._API_HEADERS,
+            schema=validate.Schema(
+                validate.parse_json(),
+                validate.any(
+                    validate.all(
+                        {"messages": validate.list(str)},
+                        validate.get(("messages", 0)),
+                        validate.transform(lambda data: (False, data)),
+                    ),
+                    validate.all(
+                        {"authorisationToken": str},
+                        validate.get("authorisationToken"),
+                        validate.transform(lambda data: (True, data)),
+                    ),
                 ),
-            )
+            ),
+        )
+
+        if not success:
+            log.error(data)
+            return
+
         return data
 
-    def login(self, email, password):
-        log.debug("Attempting login as {0}".format(email))
-        # sets some required cookies to login
-        data = self.request(
-            "POST",
-            self.login_url,
-            data=json.dumps({"id": email, "secret": password}),
-            headers={"Content-Type": "application/json"},
+    def _get_streams_content(self, content_type, content_id, token):
+        success, data = self.session.http.get(
+            self._API_URLS.get(content_type).format(content_id),
+            acceptable_status=(200, 401, 404),
+            params={"includePlaybackDetails": "URL"},
+            headers={"Authorization": f"Bearer {token}", **self._API_HEADERS},
+            schema=validate.Schema(
+                validate.parse_json(),
+                validate.any(
+                    validate.all(
+                        {"messages": validate.list(str)},
+                        validate.get(("messages", 0)),
+                        validate.transform(lambda message: (False, message)),
+                    ),
+                    validate.all(
+                        {
+                            "accessLevel": str,
+                            "id": int,
+                            "title": str,
+                            validate.optional("playerUrlCallback"): validate.any(None, validate.url()),
+                        },
+                        validate.union_get(
+                            "accessLevel",
+                            "playerUrlCallback",
+                            "id",
+                            "title",
+                        ),
+                        validate.transform(lambda data: (True, data)),
+                    ),
```

Comment:
`playerUrlCallback` is marked as optional, in addition to a potential `None` value. I guess this is because of invalid input URLs or invalid auth data where their API doesn't return a `playerUrlCallback` value. You're however not checking for a truthy `playback_url` value down below, so the follow-up HTTP GET request may have a `None` URL value. I also don't know if `id` and `title` will have proper values or values at all, considering that only `playerUrlCallback` is marked as optional.